core/src/test/scala/com/microsoft/azure/synapse/ml/nbtest/SynapseTests.scala

```diff
@@ -34,12 +34,26 @@ class SynapseTests extends TestBase {
   })
 
   val workspaceName = "mmlsparkppe"
-  val sparkPools: Array[String] = Array(
-    "e2etstspark32i1",
-    "e2etstspark32i2",
-    "e2etstspark32i3",
-    "e2etstspark32i4",
-    "e2etstspark32i5")
+  val resourceGroupName = "marhamil-mmlspark"
+  val subscriptionId = "e342c2c0-f844-4b18-9208-52c8c234c30e"
+  val expectedPoolCount = 3
+
+  println(s"SynapseTests E2E Test Suite starting. Creating ${expectedPoolCount} Spark Pools...")
+  var (sparkPools, actualPoolCount) = {
```

Comment:
avoid using var

Reply:
We alkso want to make sure that the pools are independent per build job that is run so its okay not to query for existing pools to re-use but just spin up and tear down dynamically

Reply:
That is exactly what this PR helps. We create spark pools named "tc<ts-in-millisecond>" (Eg. `tc1653515249401`) for each new e2e run and delete these at the end of graceful shutdowns to ensure per build job we have separate set of pools